advanced/Scripts/database_migration/gravity/1_to_2.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE auditlist
```

Comment:
A clearer name would be `domain_audit` or similar. In the future if we add some other kind of audit table, this name would be confusing.